versions/development/SPEC.md

```diff
@@ -802,8 +804,8 @@ Some examples of correct import resolution:
 | Root Workflow Location                                | Imported Path                      | Resolved Path                                           |
 |-------------------------------------------------------|------------------------------------|---------------------------------------------------------|
 | file://foo/bar/baz/qux.wdl                            | some/task.wdl                      | file://foo/bar/baz/some/task.wdl                        |
-| http://www.github.com/openwdl/coolwdls/myWorkflow.wdl | subworkflow.wdl                    | http://www.github.com/openwdl/coolwdls/subworkflow.wdl  |
-| http://www.github.com/openwdl/coolwdls/myWorkflow.wdl | /openwdl/otherwdls/subworkflow.wdl | http://www.github.com/openwdl/otherwdls/subworkflow.wdl |
+| `http://www.github.com/openwdl/coolwdls/myWorkflow.wdl` | subworkflow.wdl                    | `http://www.github.com/openwdl/coolwdls/subworkflow.wdl`  |
```

Comment:
I would leave these in, they're probably there to stop github from rendering these made up paths are links.

Reply:
wouldn't changing them to code blocks stop the rendering?

Reply:
I think if you're going to use code tags here to prevent the hyperlinks, you should do it to all entries in this table so it looks consistent. Or use one of the other workarounds: https://gist.github.com/alexpeattie/4729247